app/CMakeLists.txt

```diff
@@ -325,11 +325,13 @@ target_compile_definitions(firebase_app
   PRIVATE
     -DINTERNAL_EXPERIMENTAL=1
 )
+
 # firebase_app has a dependency on flatbuffers, which needs to be included.
 target_link_libraries(firebase_app
   PRIVATE
     flatbuffers
     ${LIBSECRET_LIBRARIES}
+    ${ADDITIONAL_LIBS}
```

Comment:
I don't think ADDITIONAL_LIBS is defined in this context, at least I'm not sure what this is doing?

Reply:
Ah, this was leftover from when I was trying to fix a few things, will remove.